api/nodes/serializers.py

```diff
@@ -7,12 +7,15 @@
 
 class NodeSerializer(JSONAPISerializer):
 
+    filterable_fields = ['title', 'description']
```

Comment:
Minor, but this could be a set for faster `in` checks. Of course, this will only matter if the collection is large.